components/automate-gateway/handler/license.go

```diff
@@ -75,10 +75,13 @@ func (t *LicenseServer) GetStatus(ctx context.Context,
 	}
 
 	resp := license.GetStatusResp{
-		LicenseId:      lcResp.LicenseId,
-		ConfiguredAt:   lcResp.ConfiguredAt,
-		CustomerName:   lcResp.CustomerName,
-		LicensedPeriod: (*license.GetStatusResp_DateRange)(lcResp.LicensedPeriod),
```

Comment:
these direct casts don't work any longer because you can't cast if your struct has private members (unless both are defined in the same package)